go/vt/vtctl/reparentutil/emergency_reparenter.go

```diff
@@ -518,13 +517,26 @@ func (erp *EmergencyReparenter) reparentReplicas(
 	rec := concurrency.AllErrorRecorder{}
 
 	handlePrimary := func(alias string, tablet *topodatapb.Tablet) error {
-		position, err := erp.tmc.PrimaryPosition(replCtx, tablet)
-		if err != nil {
-			return err
-		}
-		if populateReparentJournal {
+		if !intermediateReparent {
+			var position string
+			var err error
+			if ev.ShardInfo.PrimaryAlias == nil {
+				erp.logger.Infof("setting up %v as new primary for an uninitialized cluster", alias)
+				// we call InitPrimary when the PrimaryAlias in the ShardInfo is empty. This happens when we have an uninitialized cluster.
+				position, err = erp.tmc.InitPrimary(replCtx, tablet, SemiSyncAckers(opts.durability, tablet) > 0)
+			} else {
```

Comment:
Didn't we say that we should eliminate the option of using ERS to initialize a new cluster?

Reply:
I left it in there. Doesn't hurt.